example/src/main/java/org/secondbase/example/main/HelloAll.java

```diff
@@ -0,0 +1,64 @@
+package org.secondbase.example.main;
+
+import io.prometheus.client.Counter;
+import java.io.IOException;
+import org.secondbase.consul.ConsulModule;
+import org.secondbase.core.SecondBase;
+import org.secondbase.core.SecondBaseException;
+import org.secondbase.core.config.SecondBaseModule;
+import org.secondbase.flags.Flag;
+import org.secondbase.flags.Flags;
+import org.secondbase.logging.JsonLoggerModule;
+import org.secondbase.webconsole.HttpWebConsole;
+import org.secondbase.webconsole.PrometheusWebConsole;
+import org.secondbase.webconsole.widget.Widget;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class HelloAll {
+    @Flag(name="variable")
+    private static String var = "default";
+    @Flag(name="counter")
+    private static int counter = 1;
+
+    private static final Logger LOG = LoggerFactory.getLogger(HelloAll.class.getName());
+
+    public static void main(final String[] args) throws SecondBaseException, IOException {
+
+        final String[] realArgs = {
```

Comment:
As you can see, the arguments passed to the different modules are mostly the same. We should refactor a bit so that they are set once, in the SecondBase core. Then we can make the init() method of SecondBaseModule pass the SecondBase object, so that modules can optionally use the flags set there.

I'll look into that tomorrow.